### ISSUE TITLE
[4.0] Featured button [a11y]

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#33296]
Multiple issues

### Featured Article
~1. Icon uses color alone to indicate its status~
2. Screen reader only announces "Toggle Featured Status" - it does not announce if it is featured or not
3. Still uses bootstrap tooltip

![image](https://user-images.githubusercontent.com/1296369/115989351-d0762780-a5b5-11eb-960c-7ecbb75685d3.png)

### Featured Contact
~1. icon uses color alone to indicate its status~
2. Screen reader only announces "Feature Item" or "Unfeature item" - it does not announce if it is featured or not

![image](https://user-images.githubusercontent.com/1296369/115989421-2f3ba100-a5b6-11eb-932c-fb90267f1c1f.png)

## Summary
~- Update icon~
- announce both the current state and what the button will do
- consistent terminology

_updated after #33417_

**Comment by himanshu007-creator** [comment]
To solve this, an approach can be using different but similar icons for featured and unfeatured status of article.
featured state icon:- https://fontawesome.com/icons/lightbulb-on?style=solid
unfeatured state icon:- https://fontawesome.com/icons/lightbulb-slash?style=duotone

On screen , it will go like:-
> state: <_present state_>
   click to toggle

**Comment by richard67** [comment]
@himanshu007-creator The icons you are suggesting belong to the "pro" package of fontawesome, which is a paid one. We only ship the free ones, "standard", "solid" and "brands".

**Comment by chmst** [comment]
We could show no star at all if an item is not featured.

**Comment by himanshu007-creator** [comment]
> @himanshu007-creator The icons you are suggesting belong to the "pro" package of fontawesome, which is a paid one. We only ship the free ones, "standard", "solid" and "brands".

Okay, if that is the case, maybe an eye icon will do the job! Moreover , it will look quite self explanatory as well!
featured state icon-https://fontawesome.com/icons/eye?style=solid
unfeatured state icon- https://fontawesome.com/icons/low-vision?style=solid


views:-
![featured](https://user-images.githubusercontent.com/65963997/115998725-45714d80-a606-11eb-9381-e3378603de89.png)
![unfeatured](https://user-images.githubusercontent.com/65963997/115998727-47d3a780-a606-11eb-89f8-980f43372928.png)

**Comment by PhilETaylor** [comment]
Not only the featured icon - if you go to Template Styles here is the same issue with the default icon

<img width="1209" alt="Screenshot 2021-04-25 at 16 28 24" src="https://user-images.githubusercontent.com/400092/115999389-5b214b80-a5e3-11eb-960e-8732ccbe9cc1.png">

**Comment by PhilETaylor** [comment]
> Okay, if that is the case, maybe an eye icon will do the job! 

Eye is used for preview elsewhere in Joomla - Eg, the template styles screenshot [I just posted](https://github.com/joomla/joomla-cms/issues/33296#issuecomment-826342789)

**Comment by himanshu007-creator** [comment]
> > Okay, if that is the case, maybe an eye icon will do the job!
> 
> "Eye" is used for "preview" elsewhere in Joomla - Eg, the template styles screenshot [I just posted](https://github.com/joomla/joomla-cms/issues/33296#issuecomment-826342789)

 we can change the default icon here as well to a single tick icon like <a href="https://fontawesome.com/icons/check?style=solid">this</a>
A site can have many published articles, but there is only one default template for a category (site of admin). For this case we can use a  <a href="https://fontawesome.com/icons/check?style=solid">single tick</a> in front of the template selected as default and for rest templates, leave it blank.
![2021-04-25 (2)](https://user-images.githubusercontent.com/65963997/115999765-a438c600-a60a-11eb-977b-670ed82f0574.png)

**Comment by carcam** [comment]
Featured and default are different states. So having different icons for them makes total sense.

Looking for options and taking into account that an article can be featured in a specific container and it's like a windows shop, I think the store icons could do the job:

https://fontawesome.com/icons/store-slash?style=solid

https://fontawesome.com/icons/store?style=solid

For template manager, I will leave the tick for default and an empty circle for non-default items.

**Comment by chmst** [comment]
Agree that featured and default are different. 

I prefer the star as it is for feaured items, this is a common icon and users know it. 
If an item is not featured, we could use the "disc" - this is also common and known - used in menu items overview.

**Comment by alikon** [comment]
please test #33417

**Comment by richard67** [comment]
Re-opening since PR #33417 is only for parts of this issue, see https://github.com/joomla/joomla-cms/pull/33417#issuecomment-829912307 .

**Comment by brianteeman** [comment]
thanks @richard67 I have updated the original post

**Comment by chmst** [comment]
I have stopped working on the issue as @brianteeman took over.

**Comment by brianteeman** [comment]
Sorry. I didn't know you were working on it. You didn't say anything.

**Comment by richard67** [comment]
She had assigned the issue to herself, which was visible on GitHub by the "asignee".

**Comment by brianteeman** [comment]
when did we start using that? also its not very productive as only a limited few people can even use it

**Comment by richard67** [comment]
We started to use it years ago, but you are right, we used it only in rare cases.

**Comment by chmst** [comment]
Indeed it is not as easy to solve this issue, but important - because the tooltips also are different for workflow on/off.

**Comment by brianteeman** [comment]
Once again apologies for not seeing the assignment. Might be a good idea for you to review the current assignments ;)

**Comment by chmst** [comment]
no problem - now as your changes of the icon are merged I can continue.

**Comment by brianteeman** [comment]
sorry once again

**Comment by ghost** [comment]
@chmst i fear it will happen again cause the self-assign isn't shown on top:

> ![image](https://user-images.githubusercontent.com/78906980/116694412-8c24c600-a9bf-11eb-846b-ab16c62fec8b.png)

**Comment by richard67** [comment]
@sandramay0905 That's because I have remove the assignment after @chmst had asked me on Glip to do so.

**Comment by ghost** [comment]
@richard67 sorry, i thought its a general problem. Thanks for the information :-)

**Comment by chmst** [comment]
Closing as there is a PR

**Comment by richard67** [comment]
See #33557 .
